src/js/schemas/service-schema/Networking.js

```diff
@@ -9,11 +9,15 @@ const Networking = {
       title: 'Network Type',
       options: [
         {html: 'Host (Default)', id: 'host'},
-        'Bridge',
-        {html: 'Virtual Network: Dev', id: 'dev'},
+        {html: 'Bridge', id: 'bridge'},
+        {html: 'Virtual Network: Dev', id: 'user'},
```

Comment:
Right I'll change it back. I introduced this by going down another route. My bad.